scratch/copy-mk.py

```diff
@@ -57,6 +69,11 @@ def parse_args(argv: Sequence[str] | None = None) -> argparse.Namespace:
         help="Endpoint of the CBF master controller",
     )
     parser.add_argument("--name", help="Name of the subarray product to create")
+    parser.add_argument(
+        "--product-controller",
+        type=endpoint_parser(31000),  # Not the actual default port
```

Comment:
```suggestion
        type=endpoint_parser(None),
```
That will make it require a port IIRC.